doc/src/manual/style-guide.md

```diff
@@ -123,7 +123,7 @@ Types such as `Union{Function,AbstractString}` are often a sign that some design
 When creating a type such as:
 
 ```julia
-type MyType
+struct MyType
```

Comment:
it's highlighting a problem, it should be doing so in a way that's representative of when it's actually a problem